janusgraph-berkeleyje/src/test/resources/release-test-groups/bdb1

```diff
@@ -0,0 +1 @@
+org.apache.tinkerpop.gremlin.process.TraversalPerformanceTest
```

Comment:
Now that I have [CI working for the DynamoDB Storage Backend here](https://travis-ci.org/awslabs/dynamodb-titan-storage-backend/builds/237101582?utm_source=github_status&utm_medium=notification), I want to build on this experience to inform the mechanism for dividing up tests here.

Basically, considering the large number of tests we have, parameterization and the length of concatenated canonical test class names, using these kinds of files is unwieldy and potentially may cause length exceeded errors on the command line used to start tests. I would rather define test categories and use those to drive the tests.